utilities/pulumi_state_splitter/pulumi_state_splitter/model.py

```diff
@@ -22,7 +22,10 @@ class Resource(pydantic.BaseModel):
 
     model_config = pydantic.ConfigDict(extra="allow")
 
-    file_exclude: ClassVar = {"parent_resource"}
+    file_exclude: ClassVar = {
+        "parent_resource",
+        "sourcePosition",
```

Comment:
**Suggestion:** Could we add a comment documenting the reason for removing this entry?

Reply:
Done in d992dd5f484c5ce83f741f05c95193387e6d5661 .